### DISCUSSION TITLE
graphviz dependency

### DISCUSSION THREAD

**Discussion by aaronmarkham** [apache/incubator-mxnet#14918] [Q&A]
## Description
We have various documents that have graphviz as a dependency for MXNet, both as a system package and a python package, and without a version number specified.

I recently ran into an issue on the nano while working on the instructions where trying to bind Python to the libmxnet.so binary, it failed because my graphviz version exceeded the [allowed range](https://github.com/apache/incubator-mxnet/blob/de5267f43560f2f444cfa70e3e508ed79ae2acc5/python/setup.py#L33) of `graphviz<0.9.0,>=0.8.1`. I had 0.10.1. This is what any new user is going to get when following most source installation instructions. Which means they'll fail.

1. Why do we have this range?
1. If we were going to pin a version, what would it be? 0.8.4 is last release in current required range.
1. Why do we require both system and python packages? 
1. Do we even need this dependency with MXBoard being a superior alternative?

[a comment by an automated account is not shown]

**Comment by marcoabreu** [comment]
Test
